cli/fossilize_replay.cpp

```diff
@@ -1470,8 +1476,9 @@ struct ThreadedReplayer : StateCreatorInterface
 		if ((disk_pipeline_cache || opts.pipeline_stats) && device->pipeline_feedback_enabled())
 			feedback.setup_pnext(work_item);
 
-		if (vkCreateGraphicsPipelines(device->get_device(), cache, 1, work_item.create_info.graphics_create_info,
-									  nullptr, work_item.output.pipeline) == VK_SUCCESS)
+		VkResult result = vkCreateGraphicsPipelines(device->get_device(), cache, 1, work_item.create_info.graphics_create_info,
+									  nullptr, work_item.output.pipeline);
+		if (result == VK_SUCCESS)
```

Comment:
If COMPILE_REQUIRED returns false, you'll need to recompile it without the flag. Other PSOs may depend on this flag.

Reply:
Also, there are considerations to be made for GPL pipelines. It's likely that fast-linked PSOs are never placed in caches, so it's possible those have to be counted separately.